source/_patterns/01-molecules/navigation/view-selector.yaml

```diff
@@ -4,13 +4,13 @@ properties:
   sideBySideUrl:
     type: string
     minLength: 1
-  articleUrl:
+  mainUrl:
```

Comment:
I think perhaps this should be `primaryUrl`.

Reply:
All variables that contain "main" are changed to "primary"